rr_evgp/launch/localization_sim.launch

```diff
@@ -22,7 +22,7 @@
         <param name="use_odometry_yaw" value="false"/>
         <param name="wait_for_datum" value="false"/>
 
-        <remap from="/imu/data" to="/imu"/>
+        <remap from="/imu/data" to="/imu_bottom"/>
         <remap from="/gps/fix" to="/gps/fix"/>
         <remap from="/odometry/filtered" to="/odometry/filtered"/>
```

Comment:
Do these remaps end up doing anything?

Reply:
We might remap them to something else in the future (if we change localization stuff) so I think we should keep it in order to remind ourselves that this node uses it.